src/component/parser/json/converter/ProcessConverter.ts

```diff
@@ -86,12 +86,12 @@ export default class ProcessConverter extends AbstractConverter<Process> {
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   private buildFlowNodeBpmnElements(processId: string, bpmnElements: Array<any> | any, kind: ShapeBpmnElementKind): void {
     ensureIsArray(bpmnElements).forEach(bpmnElement => {
-      if (kind == ShapeBpmnElementKind.EVENT_START) {
-        // get the list of eventDefinition hold by the Start Event bpmElement
+      if (kind as Event) {
```

Comment:
to make it more verbose maybe it is better to use BpmnEventDefinition or shortly BpmnEvent as Celine suggested

Reply:
done, see b14af2b